THITD-PARTY-NOTICE.txt

```diff
@@ -0,0 +1,112 @@
+This project uses the following third party libraries:
+
+OkHttp
+------
+OkHttp is an HTTP client
+
+Project: https://square.github.io/okhttp/
+License: https://square.github.io/okhttp/#license
+
+OKHttp Logging Interceptor
+--------------------------
+An OkHttp interceptor which logs HTTP request and response data.
+
+Project: https://square.github.io/okhttp/interceptors/
+License: https://square.github.io/okhttp/#license
+
+Lombok
+------
+Project Lombok is a java library that automatically plugs into your editor and build tools, spicing up your java.
```

Comment:
Not sure how important it is, but shouldn't `java` be capitalized?

Reply:
I copy and paste from the official description from the website, I would leave it as is.